frontend/src/components/QueueTable.js

```diff
@@ -103,10 +103,18 @@ const QueueTable = observer(queueData => {
                 column="urgency"
               />
             ),
+            customSort: (a, b) => +a.urgency[1] - +b.urgency[1],
           },
           { title: "Last", field: "last" },
           { title: "UID", field: "uid" },
-          { title: "Time", field: "timeElapsed" },
+          {
+            title: "Time",
+            field: "timeElapsed",
+            // eslint-disable-next-line no-console
+            customSort: (a, b) =>
+              new Date("1970/01/01 " + a.timeElapsed) -
+              new Date("1970/01/01 " + b.timeElapsed),
```

Comment:
What kind of value do we expect `timeElapsed` to be?

Reply:
@marcelkooi timeElapsed should be the string of the hour and minutes, the format looks like "hh:mm am/pm"